vasodecanto-app/vasodecanto-app/DesignSystem/Fonts/AppFont.swift

```diff
@@ -0,0 +1,43 @@
+//  AppFont.swift
+//  vasodecanto-app
+//
+//  Created by Jade Oliveira on 29/09/25.
+
+import SwiftUI
+
+// MARK: - Estilos de Fonte do Aplicativo
+struct AppFont {
```

Comment:
Outra sugestão:

Acho que nesse contexto seria ótimo utilizar o conceito de extensão de strucs que o próprio swift nos dar para realizar o Design System. 

```suggestion
extension Font {
  struct AppFont
   // ...
```

Assim o uso poderia se dar de forma ainda mais direta e simples no código da construção de interface: 

`.font(.heeboCaption)`